### DISCUSSION TITLE
переназначение actionban/actionunban для fail2ban

### DISCUSSION THREAD

**Discussion by AlfnRU** [mikopbx/Core#288] [Ideas (Идеи)]
Возможно ли добавить изменение actionban/actionunban в fail2ban?
Можно было бы использовать для работы с граничным файрволом,.. и не пускать трафик к станции, дабы не расходовать ее не такие уж и большие ресурсы.

**Comment by boffart** [comment]
Возможно, к примеру в **cron** через кастомизацию системных файлов добавить задачу по копированию нужных файлов.
